api/apps/api/config/default.json

```diff
@@ -13,6 +13,9 @@
      },
    "concurrency": 50
    },
+  "api": {
+    "url": "http://api:3000"
```

Comment:
@hotzevzl please keep in mind to set those on deployments.

Reply:
yep, as long as they are in `README.md` and in `env.default` we're good - I think you added this one some time ago.

Reply:
for values needed in CI, we also need to add them to `.env-test-e2e` (or GH secrets if applicable) and to pass them through via `docker-compose-test-e2e.ci.yml`